src/components/PopupWithForm.js

```diff
@@ -6,6 +6,9 @@ export default class PopupWithForm extends Popup {
     this._handleFormSubmit = handleFormSubmit;
     this._form = this._popupElement.querySelector(".modal__form");
     this._inputList = this._form.querySelectorAll(".modal__input");
+    this._confirmButton = this._popupElement.querySelector(
```

Comment:
`_confirmButton` should not be here because `PopupWithForm` is for any modal with a form


* There should **not** be any code in universal functions/classes that doesn’t suit every popup.